tests/quantization/resnet18.json

```diff
@@ -0,0 +1,1061 @@
+            "qconfig": {
+                "mode": "symmetric",
+                "num_bits": 8,
+                "per_channel": true,
+                "signedness_to_force": null
+            },
+            "scopes_of_directly_quantized_operators": [
+                "ResNet/Sequential[layer4]/BasicBlock[1]/NNCFConv2d[conv2]"
+            ]
+        },
+        "69": {
+            "insertion_point": {
+                "input_port_id": null,
+                "module_scope": "ResNet/NNCFLinear[fc]",
+                "target_type": {
+                    "name": "OPERATION_WITH_WEIGHTS"
+                }
+            },
+            "qconfig": {
+                "mode": "symmetric",
+                "num_bits": 8,
+                "per_channel": true,
+                "signedness_to_force": null
+            },
+            "scopes_of_directly_quantized_operators": [
+                "ResNet/NNCFLinear[fc]"
+            ]
+        }
+    },
+    "shared_input_operation_set_groups": {
+        "0": [
+            1
+        ],
+        "1": [
+            2,
+            69
+        ],
+        "2": [
+            49,
+            3
+        ],
+        "3": [
+            51,
+            5
+        ],
+        "4": [
+            9,
+            53
+        ],
+        "5": [
+            14,
+            55
+        ],
+        "6": [
+            16,
+            15
+        ],
+        "7": [
+            18,
+            58
+        ],
+        "8": [
+            60,
+            23
+        ],
+        "9": [
+            24,
+            25
+        ],
+        "10": [
+            27,
+            63
+        ],
+        "11": [
+            32,
+            65
+        ],
+        "12": [
+            33,
+            34
+        ],
+        "13": [
+            36,
+            68
+        ],
+        "14": [
+            40,
+            50,
+            7
+        ],
+        "15": [
+            41,
+            11,
+            52
+        ],
+        "16": [
+            56,
+            42,
+            54
+        ],
+        "17": [
+            57,
+            43,
+            20
+        ],
+        "18": [
+            59,
+            44,
+            61
+        ],
+        "19": [
+            45,
+            29,
+            62
+        ],
+        "20": [
+            64,
+            66,
+            46
+        ],
+        "21": [
+            67,
+            38,
+            47
+        ]
+    },
+    "unified_scale_groups": {
+        "0": [
+            16,
+            15
+        ],
+        "1": [
+            24,
+            25
+        ],
+        "2": [
+            33,
+            34
+        ],
+        "3": [
+            40,
+            7
+        ],
+        "4": [
+            41,
+            11
+        ],
+        "5": [
+            43,
+            20
+        ],
+        "6": [
+            29,
+            45
+        ],
+        "7": [
+            38,
+            47
+        ]
+    }
+}
```

Comment:
Are these files supposed to be in the PR? If so, they need a more descriptive naming scheme.

Reply:
no, will delete this debug file